pkg/vm/engine/disttae/util.go

```diff
@@ -66,6 +66,10 @@ func getColumnMapByExpr(expr *plan.Expr, tableDef *plan.TableDef, columnMap *map
 	}
 }
 
+func GetColumnsByExpr(expr *plan.Expr, tableDef *plan.TableDef) (map[int]int, []int, int) {
```

Comment:
Do not expose these functions. If you really need to expose. Find a special directory for it.